portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,30 +22,51 @@
 import com.google.gson.Gson;
 import java.util.ArrayList;
 import java.util.List;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.common.collect.ImmutableList;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.common.collect.Streams;
+import static com.google.common.collect.ImmutableList.toImmutableList;
 
 /** Servlet that handles comment data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static final String COMMENT_NAME = "comment";
   private static final String DISPLAY_NAME = "name";
   private static final String DEFAULT_VAL = "";
-  private List<Comment> comments = new ArrayList<Comment>();
 
    @Override
-   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-     response.setContentType("application/json;");
-     Gson gson = new Gson();
-     String json = gson.toJson(comments);
-     response.getWriter().println(json);
-   }
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
```

Comment:
nit: move this down to group together with `response.getWriter().println(json);`